.github/workflows/deploy.yaml

```diff
@@ -195,3 +195,119 @@ jobs:
           export ROUTE="$(oc get route ${API} -o jsonpath='{.spec.host}')"
           echo "${API} is exposed at 'https://'${ROUTE}" 
 
+   # Deploy App and Api images in Test
+  deployTest:
```

Comment:
I think we'll need to change this a bit so it can be manually deployed as we don't want to push it to the test env every time we push to the dev env.

To keep it simple, I would suggest that we just add in a `workflow_dispatch` trigger in addition to the current `main` trigger, where you can specify the env you want to deploy to:

```
on: 
  workflow_dispatch:
    inputs:
      environmentName:
        description: 'Environment name (dev/test/prod)'     
        required: true
        default: 'dev'
```

and then conditionally trigger each job based on that:
```
  deployDev:
    name: Deploy APP to Dev environment
    if: !github.event.inputs.environmentName || github.event.inputs.environmentName === 'dev'
....
  deployTest:
    name: Deploy APP to Dev environment
    if: github.event.inputs.environmentName === 'test'


```

Reply:
For this part, I have created a github TEST environment in github which requires a Manual approval before deploying to TEST.  Would that help?
![image](https://user-images.githubusercontent.com/77285719/156027945-e44dc285-3cb9-4645-9133-09409bf8f7fa.png)